src/core/redux/selectors/selectMyKeys.js

```diff
@@ -0,0 +1,18 @@
+/*
+ * @flow
+ */
+
+import {
+  List,
+  Map,
+  Set,
+  getIn,
+} from 'immutable';
+import type { UUID } from 'lattice';
+
+import { MY_KEYS, PERMISSIONS } from '../constants';
+
+export default function selectMyKeys() {
+
+  return (state :Map) :Set<List<UUID>> => getIn(state, [PERMISSIONS, MY_KEYS]) || Set();
+}
```

Comment:
no change required, but I prefer passing the `IfNotSetValue` (in this case `Set()`) as the third parameter to `getIn()`

Reply:
It's possible that the key is set and the value is `undefined` so the "not set value" param wouldn't apply. This avoids that.